src/theme.ts

```diff
@@ -1,6 +1,8 @@
+/* used multiple times below */
+const primaryMain = '#2297DE';
 const colors = {
 	primary: {
-		main: '#2297DE',
+		main: primaryMain,
```

Comment:
Was this by accident? The issue & PR title says that we should use `Primary.light` instead of `main`.
cc: @konmouz in case we prefer to go with `Primary.main`

Reply:
Nope, my mistake o.O Ooops.